.changeset/quiet-numbers-scream.md

```diff
@@ -0,0 +1,8 @@
+---
+"@cloudoperators/juno-url-state-provider": minor
```

Comment:
- @cloudoperators/juno-url-state-provider
  -  Should we still release the package or would it make more sense to make it private?

Reply:
We can make it private but I don't see a need for that because we were already publishing `url-state-provider` to `npm` and in future we need it as a public package so that the third party app developers can utilize this package.

Reply:
yes sure!! but now we are encoding different then before and I could break deep links to someone using the actual version...